src/nvim/tui/tui.c

```diff
@@ -79,6 +79,7 @@ struct TUIData {
   Loop *loop;
   unibi_var_t params[9];
   char buf[OUTBUF_SIZE];
+  char *buf_to_flush;  ///< Non-null to flush this instead of buf[]
```

Comment:
this took me a few seconds to comprehend
```suggestion
  char *buf_to_flush;  ///< If non-null, flush this instead of buf[].
```